lms/templates/commerce/checkout_error.html

```diff
@@ -0,0 +1,12 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("Checkout Error")}</%block>
+
+
+<section class="container">
+    <h1>${_("Checkout Error")}</h1>
+    ${ _(u"An error occurred while processing your payment; you have not been charged. Please try again. If the issue persists, contact {email}.").format(
```

Comment:
@zubair-arbi the message is correct given the scenario we want to use this page in: a user has *authorized* payment, but we haven't been able to *execute* the payment and actually collect the money. You can verify this yourself by checking the balance on your PayPal account, forcing an exception to be raised in the PayPal payment handling view when attempting to pay for a course seat, then checking the balance again. When I tried this, I observed that the balance was unchanged, indicating that the user wasn't actually charged.

@srpearce could you please review this message? We want to reassure the user that they haven't been charged, and that they should try submitting their payment again.